#!/usr/bin/python3
def raise_exception_msg(e=""):
    raise NameError(e)
